pkg/editing/issue.go

```diff
@@ -14,3 +14,12 @@ func (i *Issue) AddField(key string, val interface{}) {
 
 	i.Fields[key] = val
 }
+
+func (i *Issue) HasParent() (bool, string) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method Issue.HasParent should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4777494)